src/core/lucky_rocket.h

```diff
@@ -4,16 +4,12 @@
 
 class LuckyRocket :	public Weapon
 {
+public:
```

Comment:
the ctor was private intentionally, so only the itemfactory was able to instantiate a lucky rocket